core/src/test/java/io/questdb/test/griffin/ExplainPlanTest.java

```diff
@@ -2415,7 +2415,7 @@ public void testHashLeftJoin1() throws Exception {
 
```

Comment:
minor: it'd be good to change EqIntStrCFunctionFactory.Func.toPlan to show null instead of NaN .